.vscode/tasks.json

```diff
@@ -1,49 +1,28 @@
-// See https://go.microsoft.com/fwlink/?LinkId=733558
-// for the documentation about the tasks.json format
 {
-    "version": "2.0.0",
-    "tasks": [
-        {
-            "type": "npm",
-            "script": "watch",
-            "problemMatcher": "$tsc-watch",
-            "isBackground": true,
-            "group": {
-                "kind": "build",
-                "isDefault": true
-            },
-            "dependsOn": ["Build Explorer"]
-        },
-        {
-            "label": "Install node modules",
-            "type": "npm",
-            "script": "installNodeModules",
-            "problemMatcher": [],
-            "group": "build"
-        },
-        {
-            "label": "Build Explorer",
-            "type": "npm",
-            "script": "buildExplorer",
-            "problemMatcher": "$tsc-watch",
-            "group": "build",
-            "dependsOn": ["Install node modules"]
-        },
-        {
-            "label": "Build Explorer Watch",
-            "isBackground": true,
-            "type": "npm",
-            "script": "buildExplorerWatch",
-            "problemMatcher": "$tsc-watch",
-            "group": "build",
-        },
-        {
-            "label": "tslint",
-            "type": "npm",
-            "script": "lint",
-            "problemMatcher": {
-                "base": "$tslint5"
-            }
-        }
-    ]
-}
\ No newline at end of file
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "start",
```

Comment:
>"script": "start", [](start = 3, length = 18)

Have the script depend on an "npm install" task so you can clone the repo and just press F5. It will also cause npm install if a developer does a pull as well (added benefit).

Reply:
The initial motivation was to conditionally not spend time doing `npm install` if `node_modules` was already present to save some dev loop time. We discussed offline this is no big deal, which removes the need for the script, and that should satisfy this comment.